pkg/apis/core/types.go

```diff
@@ -2788,11 +2788,17 @@ type ContainerStatus struct {
 }
 
 type ResourceStatus struct {
+	// Name of the resource. Must be unique within the pod and in case of non-DRA resource, match one of the resources from the pod spec.
+	// For DRA resources, the value must be claim:claim_name/request.
+	// The claim_name must match one of the claims from resourceClaims field in the podSpec.
+	// +required
 	Name ResourceName
-	// List of unique Resources health. Each element in the list contains a unique resource ID and resource health.
+	// List of unique Resources health. Each element in the list contains an unique resource ID and resource health.
```

Comment:
don't capitalize Resource unless it's another type, but if so, capitalize above?

Reply:
improved wording